src/deinflect.ts

```diff
@@ -153,17 +158,17 @@ const deinflectRuleData: Array<[string, string, number, number]> = [
   ['こうたり', 'こう', 640, DeinflectReason.Tari],
   ['こさせる', 'くる', 2049, DeinflectReason.Causative],
   ['こられる', 'くる', 2049, DeinflectReason.PotentialOrPassive],
-  ['しすぎる', 'す', 513, DeinflectReason.Sugiru],
+  ['しすぎる', 'す', 4609, DeinflectReason.Sugiru],
   ['しすぎる', 'する', 4097, DeinflectReason.Sugiru],
-  ['しちゃう', 'す', 514, DeinflectReason.Chau],
+  ['しちゃう', 'す', 4609, DeinflectReason.Chau],
```

Comment:
Should this be 4610?

i.e. we used to have:
to: WordType.Godan
from: WordType.Godan
= 00000010 00000010
= 514

But now we have:
to: WordType.Godan or WordType.SuruVerb
from: WordType.Godan
= 00010010 00000010
= 4610